src/calculation/ClauseResultsBuilder.ts

```diff
@@ -18,6 +18,15 @@ import {
 } from '../types/Calculator';
 import { UnexpectedProperty } from '../types/errors/CustomErrors';
 
+// code system, search parameters, and code paths for pulling pretty information from resources
+import _systemMap from '../code-system/system-map.json';
+const systemMap = _systemMap as Record<string, string>;
```

Comment:
We import the JSON file directly and then cast it to `Record<string, string>`. Would it be easier to just make it `systemMap.ts` or something and export a `const` that is the proper `Record` type?